### PR TITLE
Add IoT-J Paper

### PR DESCRIPTION
**Pull request by gavinatthu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1805,7 +1805,9 @@ IEEE Access, 2020.
 - <a name="Banerjee20arxiv"></a>Banerjee, S., Wang, Z. W., Chopp, H. H., Cossairt, O., Katsaggelos, A.,  
 *[Quadtree Driven Lossy Event Compression](https://arxiv.org/pdf/2005.00974.pdf)*,  
 arXiv 2020. [PDF](https://arxiv.org/pdf/2005.00974.pdf)
-
+- <a name="Khan21IoT-J"></a>N. Khan, K. Iqbal and M. G. Martini,
```
Comment (guillermogb) [inline README.md:1808]: Thanks. Could you...
- format the names as in a previous entry: last name, initial of name, etc.
- use Khan21jiot as key? (from the letters in the DOI: 10.1109/JIOT.2020.3007866)

File: README.md [hunk 2]
```diff
@@ -1805,7 +1805,9 @@ IEEE Access, 2020.
 - <a name="Banerjee20arxiv"></a>Banerjee, S., Wang, Z. W., Chopp, H. H., Cossairt, O., Katsaggelos, A.,  
 *[Quadtree Driven Lossy Event Compression](https://arxiv.org/pdf/2005.00974.pdf)*,  
 arXiv 2020. [PDF](https://arxiv.org/pdf/2005.00974.pdf)
-
+- <a name="Khan21IoT-J"></a>N. Khan, K. Iqbal and M. G. Martini,  
+*[Time-Aggregation-Based Lossless Video Encoding for Neuromorphic Vision Sensor Data](https://ieeexplore.ieee.org/document/9136663)*,
```
Comment (guillermogb) [inline README.md:1809]: We prefer to replace the url link with the DOI: https://doi.org/10.1109/JIOT.2020.3007866

**Review by guillermogb — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (gavinatthu) [reply README.md]: Thanks for reply, I've changed them.

File: README.md (hunk 2 above)
Comment (gavinatthu) [reply README.md]: I've changed the url link

**Review by gavinatthu — COMMENTED** [review]
Thanks for the review

File: README.md (hunk 1 above)
Comment (guillermogb) [reply README.md]: Thanks! If you have a publicly accessible pdf (preprint or similar), it would be good to provide it, too.

**Review by guillermogb — APPROVED** [review]

**Pull request merged**